MODULAR.md

```diff
@@ -12,9 +12,7 @@ New config files must follow the same format, listing one module name per line w
 
 ### Adding new modules into the build process
 
-Adding a custom module into the Arkouda buld process is treated the same way as a regular module: added in a single line in the server configuration file.
-
-Currently, modules will only be found by the Arkouda build if located in the `src/` directory.
+The only unique aspect of adding modules from other directories into the Arkouda build process is that a complete path must be specified for all modules not found in the Arkouda `src/` directory. This would actually be done by adding a line such as `/Users/test-user/path/to/mod` to `ServerModules.cfg` to add the module `mod.chpl` to the Arkouda build. Aside from that, adding a custom module into the Arkouda buld process is treated the same way as a regular module: added in a single line in the server configuration file.
```

Comment:
very small note:
I think it's clearer here to say "This is done by" instead of "This would actually be done by"

Reply:
Makes sense, thanks for pointing that out; added.